benefits/static/css/styles.css

```diff
@@ -706,19 +706,6 @@ a.card:focus-visible {
   font-size: var(--modal-title-font-size);
 }
 
-.modal-trigger--link {
-  background: transparent;
-  border: none;
-  color: var(--primary-color);
-  font-family: var(--bs-body-font-family);
-  font-weight: var(--bold-font-weight);
-  letter-spacing: var(--body-letter-spacing);
-  padding: 0;
-  margin-right: -5px;
-  text-decoration: underline;
-  text-decoration-style: solid;
-}
```

Comment:
These are no longer necessary b/c we're using an `<a>` element for the modal trigger. These classes were only overriding `<button>` defaults and adding `<a>` defaults.